src/main/java/io/anserini/server/Controller.java

```diff
@@ -3,16 +3,28 @@
 import java.util.List;
 
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 @RequestMapping(path = "/api")
 public class Controller {
-  @RequestMapping(method = RequestMethod.GET, path = "/query/{query}")
-  public List<QueryResult> search(@PathVariable("query") String query) {
-    SearchService searchService = new SearchService("msmarco-v1-passage");
+
+  private static final String DEFAULT_COLLECTION = "msmarco-v1-passage";
+
+  @RequestMapping(method = RequestMethod.GET, path = {"/collection/{collection}/search", "/search"})
+  public List<QueryResult> search(
+      @PathVariable(value = "collection", required = false) String collection, 
```

Comment:
I use a line width of 120 characters, so can go on previous line?
I care more about "looking right" than observing strict limits.